apps/pie-docs/test/snapshots/expected-routes.snapshot.json

```diff
@@ -81,36 +81,28 @@
     "engineers/getting-started/structure",
     "engineers/guidelines/browser-support",
     "engineers/guidelines/overview",
-    "foundations/colour/overview",
```

Comment:
Are we not expecting there to be a `foundations/colour` entry here? Or were these pages that didn't exist but we still had screenshots anyway?

Reply:
updated :)